FhirToDataLake/docs/Deploy-DicomToDatalake.md

```diff
@@ -0,0 +1,124 @@
+# DICOM to Synapse Sync Agent
+
+DICOM to Synapse Sync Agent enables you to perform Analytics and Machine Learning on DICOM metadata by moving DICOM metadata to Azure Data Lake in near real time and making it available to a Synapse workspace.
+
+It is an [Azure Container App](https://learn.microsoft.com/en-us/azure/container-apps/?ocid=AID3042118) that extracts data from a DICOM server using DICOM [Search](https://learn.microsoft.com/en-us/azure/healthcare-apis/dicom/dicom-services-conformance-statement#search-qido-rs) APIs, converts it to hierarchical Parquet files, and writes it to Azure Data Lake in near real time. This solution also contains a script to create External Table in Synapse Serverless SQL pool pointing to the DICOM Parquet files. For more information about External Tables and Views.
```

Comment:
Instead of search API, we are using changefeed to query metadata from DICOM service. We can refer to the document [here](https://learn.microsoft.com/en-us/azure/healthcare-apis/dicom/dicom-change-feed-overview).

Reply:
The last sentence looks referring to another data mapping document. We can add it later and remove the last sentence or add it now and complete the last sentence.